<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>南京鲛腾SFTP上传工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>531</width>
      <height>380</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>SFTP上传地址：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ip"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>端口：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="port"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>用户名：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="user_name"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="password_horizontal">
         <item>
          <widget class="QLabel" name="password_label">
           <property name="text">
            <string>密码：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pass_word"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>扫描路径：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="scan_path"/>
         </item>
         <item>
          <widget class="QPushButton" name="scanPath_file">
           <property name="text">
            <string>file</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>上传文件备份路径：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="scan_path_backup"/>
         </item>
         <item>
          <widget class="QPushButton" name="scanPathBackup_file">
           <property name="text">
            <string>file</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>SFTP上传路径：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sftp_path">
           <property name="text">
            <string>/</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>循环扫描时间(分钟)：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="circle_time"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="use_private">
         <property name="text">
          <string>使用秘钥</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="public_horizontal">
         <item>
          <widget class="QLabel" name="public_key_label">
           <property name="text">
            <string>公钥：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="public_key">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="public_file">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>file</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="private_horizontal">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>秘钥：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="private_key">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="private_file">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>file</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>密码(暂不使用)：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pass_phrase">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="save">
           <property name="text">
            <string>保存配置</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="operate_click">
           <property name="text">
            <string>启动</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="debug_output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>531</width>
      <height>101</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="overwriteMode">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>557</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
